homeassistant/components/livebox/device_tracker.py

```diff
@@ -0,0 +1,64 @@
+"""Support for Livebox devices."""
+from collections import namedtuple
+import logging
+
+from homeassistant.components.device_tracker import DeviceScanner
```

Comment:
Import and use the `ScannerEntity` instead.

https://github.com/home-assistant/home-assistant/blob/50cec91cf018bdb6a094bc1a993b80836543e96d/homeassistant/components/device_tracker/config_entry.py#L120

It has a different interface than the legacy device scanner.

Reply:
OK i rewrite Device Tracker module